tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -86,6 +86,7 @@ Standard output:
 @uifabric/merge-styles:  PASS  src/Stylesheet.test.ts
 @uifabric/merge-styles:  PASS  src/extractStyleParts.test.ts
 @uifabric/merge-styles:  PASS  src/concatStyleSetsWithProps.test.ts
+@uifabric/merge-styles:  PASS  src/fontFace.test.ts
```

Comment:
[This file](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/merge-styles/src/fontFace.test.ts) isn't blinking in and out of existence or anything... I think something is failing to determinedly always load it... If we see this line get added/removed from the baselines again, I'd be concerned. Though it might be a bug in `jest` or `webpack` and not us. :S